config/v1alpha1/types_image_policy.go

```diff
@@ -143,6 +151,39 @@ type PolicyFulcioSubject struct {
 	SignedEmail string `json:"signedEmail"`
 }
 
+// PKI defines the root of trust based on Root CA(s) and corresponding intermediate certificates.
+// +openshift:enable:FeatureGate=SigstoreImageVerificationPKI
+type PKI struct {
+	// caRootsData contains base64-encoded data of a certificate bundle PEM file, which contains one or more CA roots in the PEM format.
```

Comment:
Can you have more than one root CA? Is this why we need intermediates as a separate list?

Reply:
yes, the caRootsData can contain more than one root CA.